.github/workflows/publish.yaml

```diff
@@ -0,0 +1,46 @@
+name: publish
+
+on:
+  push:
+    tags:
```

Comment:
Looks good. Have you tested it in your fork?

Reply:
yep, I added a screenshot for the action result.